_posts/2020-08-31-final-report-yuxuan.md

```diff
@@ -133,18 +138,18 @@ dask.compute(D_1, D_2, D_3, D_4...)  #  D_x as an individual analysis job.
 
 ## Conclusions
 
-These three months of GSoC project have been fun and fruitful. Thanks to all my mentors, @IAlibay, @fiona-naughton, @orbeckst, and @richardjgowers, (also @kain88-de and other developers) for their help, reviewing PRs, and providing great advice for this project. Now MDAnalaysis has a more streamlined way to build parallel code for the trajectory analysis. What's left to do is reaching a consensus on what the future parallel analysis API should look like, minimizing the memory usage and time for the serialization process, and conducting conclusive benchmarks on different parallel engines 
+These three months of GSoC project have been fun and fruitful. Thanks to all my mentors, @IAlibay, @fiona-naughton, @orbeckst, and @richardjgowers, (also @kain88-de and other developers) for their help, reviewing PRs, and providing great advice for this project. Now MDAnalysis has a more streamlined way to build parallel code for the trajectory analysis. What's left to do is reaching a consensus on what the future parallel analysis API should look like, minimizing the memory usage and time for the serialization process, and conducting conclusive benchmarks on different parallel engines 
 
 ## Appendix
 
 ### Benchmark
-The benchmark below was run with four AMD Opteron 6274 (64 cores in total). The test trajectories (9000 frames, 111815 atoms) were available at DOI [10.6084/m9.figshare.8202149](https://figshare.com/articles/Raw_data/6025748).
+The benchmark below was run with four AMD Opteron 6274 (64 cores in total) on a single node. The test trajectories (9000 frames, 111815 atoms) were available at the [YiiP Membrane Protein Equilibrium Dataset](https://www.mdanalysis.org/MDAnalysisData/yiip_equilibrium.html).
 
-Three test cases were conducted under the latest MDAnalysis and PMDA code (PR #136)---RMSD analysis,  a more compute-intensive RDF analysis, and the newly supported RMSD analysis with on-the-fly transformation (fit-rot-trans transformation in this case). All these cases show a strong scaling performance before 16 cores. 
+Three test cases were conducted under the latest MDAnalysis and PMDA code (PR #136)---RMSD analysis, a more compute-intensive RDF analysis, and the newly supported RMSD analysis with on-the-fly transformation (fit-rot-trans transformation in this case). All these cases show a strong scaling performance before 16 cores. 
 
 ![Benchmark](/public/images/final_report_yuxuan/benchmarking.png)
 
-(Black: serial analysis; Green: colored by the number of blocks (from 1-64), darker means more blocks)
+(Black dash line: ideal strong scaling performance; Blue: parallel performance (the number of blocks being splitted ie equal to the number of cores being used)
```

Comment:
Interesting – what kind of device were trajectories read from? SSD, local disk, ... ?

I don't understand the terrible performance of the OTF fitting but that's a problem for another day.

Reply:
It's a ceph file system in our local cluster...(I am not sure about the I/O speed). We should do some more conclusive benchmarks with different systems later ofc.

In terms of OTF, I think it's due to transformation (lib.transformations) itself can use all cores.

Reply:
Maybe such coordinates transformations could be dumped into GPUs to avoid competition?...in the long run.